node/src/exchange.rs

```diff
@@ -21,11 +21,13 @@ use tracing::instrument;
 mod client;
 mod server;
 mod utils;
+pub use utils::ExtendedHeaderExt;
```

Comment:
why this?

Reply:
since utils was private I had to pry that out somehow :P
`gen_extended_header` and `gen_filled_store` live in `node::store::tests` now, but it isn't ideal as it's used in other tests too.. Should we spin out `test-utils` or similar for non-localised test utils?

Reply:
Yes we can have a `test_utils.rs`. I was also thinking about it.

Reply:
Created `test_utils` and moved this and `gen_extended_header` there.